tests/golden/builtin/builtin/builtin/01_keycloak_helmchart/postgresql/templates/primary/statefulset.yaml

```diff
@@ -7,17 +7,17 @@ metadata:
     app.kubernetes.io/instance: builtin
     app.kubernetes.io/managed-by: commodore
     app.kubernetes.io/name: keycloak
-    helm.sh/chart: postgresql-10.16.2
+    helm.sh/chart: postgresql-11.6.15
   name: keycloak-postgresql
   namespace: syn-builtin
 spec:
   replicas: 1
   selector:
     matchLabels:
+      app.kubernetes.io/component: primary
       app.kubernetes.io/instance: keycloak
       app.kubernetes.io/name: postgresql
-      role: primary
-  serviceName: keycloak-postgresql-headless
+  serviceName: keycloak-postgresql-hl
```

Comment:
is there a benefit to rename the service?

Reply:
I don't know, because this is inherited by the helm chart change and therefore we have no influence on that.